[quantization] JIT trace testing

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23987 [quantization] JIT trace testing**
* #24298 [JIT] Add the ability to compile exports on traced modules
* #24211 [quantization] equal() for QuantizedCPU

Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
[quantization] JIT trace testing

Commit:
Update on "[quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

zafartahirov commented on test/common_quantization.py:
```diff
@@ -123,6 +123,15 @@ def checkScriptable(self, orig_mod, calib_data, check_save_load=False):
 
+        # Use first calib_data entry as trace input
+        #
+        # TODO: Trace checking is blocked on this issue:
```
Create an issue, and reference this there.

jamesr66a replied:
There's already an issue listed right below... No need to create a duplicate and clog up the issue tracker

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[WIP][quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)

Commit:
Update on "[quantization] JIT trace testing"


Differential Revision: [D16744208](https://our.internmc.facebook.com/intern/diff/D16744208)
